tests/lib/schema_validation_test.py

```diff
@@ -38,13 +38,17 @@
             'link-speed': 1000,
```

Comment:
please add one test for ethernet and one for common (bond or vlan)

Reply:
Ok, I will do it in the next commit.